### ISSUE TITLE
Assigning damage to blocker is wrong?

### ISSUE THREAD

**Issue by Nymra** [Card-Forge/forge#5391]
Note: might be user error in case I missed an option or button somewhere. I checked a dozen times and cannot find anything, please show me in case this exists.

**_ISSUE:_**

When I attack player X with my 10/10
and he blocks with a 1/1, a 2/2 and an 8/8,

**In FORGE it works like this:**
Player X (defending player) decides and can say his 1/1 and the 2/2 get damage first or block first and get damage first because of that, so that the 8/8 (blocking "last") will only get 7 damage and so non-lethal.

**But MTG rules say this:**
I (the attacking player) decide and I can asign the damage to the 8/8 first to kill it and then the 2/2, leaving the 1/1 alive?

Basically Forge plays like all defending creatures have banding...

**Comment by kevlahnota** [comment]
@Nymra Can you be more specific or tell what cards are in play, any screenshots or video? Seems to work fine here if you watch the video example on latest master:

https://github.com/Card-Forge/forge/assets/9781539/b5e40630-4edf-47c8-a207-99dd13d24b0d

**Comment by tehdiplomat** [comment]
Any chance `Defensive Formation` was in play? Noone has changed anything with this any time recently. And its been working great for many years. Maybe you quick clicked through the ordering popup shown in the video above?

**Comment by Nymra** [comment]
> Any chance `Defensive Formation` was in play? Noone has changed anything with this any time recently. And its been working great for many years. Maybe you quick clicked through the ordering popup shown in the video above?

hey
thx for the responses.

No, Defensive Formation or similar is not in play.
I try to catch that window from the video, there is a chance I really missed that.
